### ISSUE TITLE
[ENHANCEMENT] Enable isolateEmulatorThread through UI

### ISSUE THREAD

**Issue by brandboat** [yasker/vm#6672]
**Is your enhancement related to a problem? Please describe.**
per discussed in https://github.com/harvester/terraform-provider-harvester/pull/105#issuecomment-2379206425,
> But one thing to consider is that a feature that goes hand-in-hand with CPU pinning is isolating the Qemu threads: https://kubevirt.io/user-guide/compute/dedicated_cpu_resources/#requesting-dedicated-cpu-for-qemu-emulator
> 
> This is an additional flag that can be set to allocate an additional physical CPU just for the purpose of running the emulator and (depending on the ioThreadsPolicy) the io threads. This can have an impact on the latency of the remaining CPUs, that are assigned to the VM.
> 
> Perhaps this flag should also be made available, to fully support workloads that benefit from CPU pinning?

It would be nice to add additional attribute `isolateEmulatorThread` to VirtualMachine CR through UI.

**Describe the solution you'd like**
Add a button next to `Enable CPU Pinning`, named `Isolate Emulator Thread `, and notify user that this option can be enabled only when VM CPU pinning enabled. And also notify user that this option acquire one more CPU.
![image](https://github.com/user-attachments/assets/51828f52-1664-454e-8e0a-8ad22da7e615)

**Describe alternatives you've considered**
N/A

**Additional context**
Of course, user can also modify Virtual Machine YAML to achieve the same thing.

**Comment by innobead** [comment]
@brandboat is this UI effort only?

**Comment by brandboat** [comment]
@innobead, Yes, I think there is no backend change here. One thing to notice is that enabling `isolateEmulatorThread` will make VM acquire one more CPU, e.g., user set 4 cpus, but the vm actually takes 5 cpus. And this should be mentioned in the `isolateEmulatorThread` check box info.

**Comment by Vicente-Cheng** [comment]
Consider the UI resource, move to v1.10.0 first.

**Comment by Vicente-Cheng** [comment]
UI effort only; we already support it in the Terraform provider. Moved to v1.11.0
